server/market/orderrouter.go

```diff
@@ -142,9 +133,9 @@ func NewOrderRouter(cfg *OrderRouterConfig) *OrderRouter {
 		tunnels:  cfg.Markets,
 		mbBuffer: cfg.MarketBuyBuffer,
 	}
-	cfg.AuthManager.Route(msgjson.LimitRoute, router.handleLimit)
-	cfg.AuthManager.Route(msgjson.MarketRoute, router.handleMarket)
-	cfg.AuthManager.Route(msgjson.CancelRoute, router.handleCancel)
+	cfg.AuthManager.RegisterMethod(msgjson.LimitRoute, router.handleLimit)
+	cfg.AuthManager.RegisterMethod(msgjson.MarketRoute, router.handleMarket)
+	cfg.AuthManager.RegisterMethod(msgjson.CancelRoute, router.handleCancel)
```

Comment:
I'm still not quite sure what's going on here. The `AuthManager` (`auth` package) has this method, and it's called `Route`. `RegisterMethod` is a holdover in some packages from when we were using JSON-RPC, which uses "methods". I was planning on changing them all to `Route`, but if you don't like `Route`, then `RegisterRoute` seems like it would be fitting. It'll need to be changed in the `auth` package's `AuthManager` too.

Reply:
No, `Route` is fine.  I must have misunderstood our previous discussion about this method name.  I'm just trying to reconcile `swap.AuthManager` and `auth.AuthManager`.  These will be an instance of the same concrete type, correct?

Reply:
Although if NewMarket just takes a Swapper, then it's likely a moot point.

Reply:
It's probably not helpful that the interfaces and the concrete type all have the same name, but yes. the `AuthManager` from `auth` is intended to satisfy the interfaces defined in `market/orderrouter.go` and `swap/swap.go`, which were written first. I had just forgotten to update the name of the `RegisterRoute` methods when the switch was made from JSON-RPC. It would've been obvious when everything was put together, but pretty confusing, for sure.